lib/api/controllers/serverController.js

```diff
@@ -25,10 +25,7 @@ const
   BaseController = require('./controller'),
   Bluebird = require('bluebird'),
   os = require('os'),
-  {
-    ServiceUnavailableError,
-    ExternalServiceError
-  } = require('kuzzle-common-objects').errors;
+  { ServiceUnavailableError } = require('kuzzle-common-objects').errors;
```

Comment:
This should not be there anymore.

I guess it's still here because of these kind of lines:

```
request.setError(new ServiceUnavailableError(error));
```

That means that `kuzzle.throw` is not sufficient for our needs, and I guess we need to expose another kuzzle function, something like `kuzzle.getError`, which takes exactly the same arguments than "throw" but returns the built error object instead of throwing it

if you do add this function, then `kuzzle.throw` would be simply that:
```
throw (...args) {
  throw this.getError(...args);
}
```

Reply:
I don't understand what this getter has to do with the current discussion?

Reply:
The problem is -> How to know in `kuzzle.throw` when get the built error or get the error which has been set by `setError` ?
Or i didnt understand the point of this